docs/erc1155-skills.md

```diff
@@ -215,20 +215,23 @@ config:
   public_uri: 127.0.0.1:9001
 ```
 
-Or, run this command:
+where `SOME_ADDRESS` is the output
+of `aea get-multiaddress cosmos --connection` in the `erc1155_deployer` project.
+The output will be something like `/dns4/127.0.0.1/tcp/9000/p2p/16Uiu2HAm2JPsUX1Su59YVDXJQizYkNSe8JCusqRpLeeTbvY76fE5`.
+
+
+Or, run these commands:
 ``` bash
 aea config set --type dict vendor.fetchai.connections.p2p_libp2p.config \
 '{
   "delegate_uri": "127.0.0.1:11001",
-  "entry_peers": ["SOME_ADDRESS"],
+  "entry_peers": ["'$(cd ../erc1155_deployer && aea get-multiaddress cosmos -c -i fetchai/p2p_libp2p:0.12.0 -u public_uri)'"],
```

Comment:
This doesn't work on Windows though

Reply:
Just keep it as a separate step in the guide?